plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/index.tsx

```diff
@@ -12,27 +12,10 @@ import metadata from './block.json';
 import AddToCartOptionsEdit from './edit';
 import './style.scss';
 
-const blockSettings = {
-	edit: AddToCartOptionsEdit,
-	icon: {
-		src: (
-			<Icon
-				icon={ button }
-				className="wc-block-editor-components-block-icon"
-			/>
-		),
-	},
-	ancestor: [ 'woocommerce/single-product' ],
-	save() {
-		return null;
-	},
-};
-
 if ( isExperimentalBlocksEnabled() ) {
-	registerBlockSingleProductTemplate( {
-		blockName: metadata.name,
-		blockMetadata: metadata,
-		blockSettings,
-		isAvailableOnPostEditor: true,
+	registerBlockType( metadata, {
```

Comment:
I think that if we don’t register the block this way, it won’t be available in the `Single Product` template.

![Screenshot 2024-12-02 at 12 27 14 PM](https://github.com/user-attachments/assets/d879ef61-5863-4e10-8eb5-ab56e34a79ca)

I'm okay with replacing the registering method, but as far as I understand, the `Add to Cart with Options` block should also be available for use in the `Single Product` template.

Reply:
it should be there, in fact, it's described in the testing instructions

Reply:
Yes, you're right, Nice catch, Fernando. Let me update the code.